新一代 Wechaty Puppet 服务：PadLink

padlink opened the pull request:
<!-- ELLIPSIS_HIDDEN -->



> [!IMPORTANT]
> Adds a blog post introducing PadLink, a new Wechaty Puppet service, highlighting its features and inviting developers to the public beta.
> 
>   - **New Blog Post**:
>     - Adds `2025-07-11-wechaty-puppet-padlink.md` introducing PadLink, a new Wechaty Puppet service.
>   - **Content Highlights**:
>     - Describes PadLink's high availability, data security, easy integration, and high performance.
>     - Discusses use cases like personal assistants, enterprise automation, and content delivery.
>     - Invites developers to join the public beta and provide feedback.
> 
> <sup>This description was created by </sup>[<img alt="Ellipsis" src="https://img.shields.io/badge/Ellipsis-blue?color=175173">](https://www.ellipsis.dev?ref=wechaty%2Fjekyll&utm_source=github&utm_medium=referral)<sup> for 16afbf257b99a6102549257727444f91f7e53656. You can [customize](https://app.ellipsis.dev/wechaty/settings/summaries) this summary. It will automatically update as commits are pushed.</sup>

<!-- ELLIPSIS_HIDDEN -->

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Documentation**
  * Added a new blog post announcing PadLink, a next-generation Wechaty Puppet service. The post highlights its key features, use cases, and invites developers to join the public beta and provide feedback.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/wechaty/jekyll?pullRequest=196) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/wechaty/jekyll?pullRequest=196) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/wechaty/jekyll?pullRequest=196) it.</sub>

huan commented:
Can you sign the CLA and make sure the CI turns green before I can review this PR?

Thank you!

padlink commented on jekyll/_posts/2025-07-11-wechaty-puppet-padlink.md:
```diff
@@ -0,0 +1,46 @@
+---
+title: "新一代 Wechaty Puppet 服务：PadLink"
+author: padlink
+layout: post
+date: 2025-07-11
+image: /assets/2025/padlink/logo.webp
+categories: announcement
```
Hi, thank you for the feedback!
I've double-checked that the hero image file assets/2025/padlink/logo.webp has already been added in this PR.
It's included in my commit and exists in the correct path with the correct filename and format.
You can also find it in the Files changed tab.

If the image is still being flagged as missing, it might be due to a caching issue or the CI not picking up static asset files correctly.
Please help re-trigger the CI or let me know if anything else needs to be adjusted.

Thanks again!

padlink commented:
> **Actionable comments posted: 0**
> 
> 🧹 Nitpick comments (1)
> 📜 Review details
> **Configuration used: CodeRabbit UI** **Review profile: CHILL** **Plan: Pro**
> 
> 📥 Commits
> 📒 Files selected for processing (1)
> 🧰 Additional context used

Thanks for the suggestion.
After consideration, the original phrasing “长期的开发和运维过程中” is more natural and appropriate for this formal context.
We’ll keep it as is for clarity and style.

padlink commented:
Hi @huan , I've resolved the review suggestions and confirmed the hero image is present.
Let me know if there's anything else needed, otherwise this PR is ready for your review. Thanks!

huan commented:
The blog structure works well, but the content feels too brief.

Could you expand the post by adding more stories about this puppet and include example code so developers can easily understand how to use it?

It would also be great to add some relevant links so interested readers can learn more or explore further—right now, the post feels empty without any references.

Thanks!

padlink commented:
@huan I have addressed the feedback and enhanced the PadLink announcement with background story, practical examples, and useful links.  The CI workflow is still pending and requires maintainer approval to proceed.  Looking forward to your review.  Thank you!

huan commented:
It has not passed the CI. Please fix it and then it's ready to be merged.
